test/unittest/layers/meson.build

```diff
@@ -26,6 +50,8 @@ exe = executable(
   'unittest_layers', test_target,
   dependencies: [
     nntrainer_test_main_deps,
+    nntrainer_layer_common_standalone_tests_dep,
+    nntrainer_layer_common_dependent_tests_dep
```

Comment:
How about we only have standalone test here, as this is an example for the layer developers?

Reply:
you mean create two `unittest_layers` test targets?

Reply:
Morelike not including dependent test, to think in a way this folder is in a separate repo, but it's super minor, so up to you :)

Reply:
My thought is to package nntrainer_layer_common_standalone_tests_lib with the devel_tests package, so the developer can test with it, like the application in this test. This is the unittest of the internal tests, which have extra unittests (dependent ones :))

Reply:
Cool